modules/stochastic_tools/src/actions/LoadSurrogateDataAction.C

```diff
@@ -29,15 +29,18 @@ LoadSurrogateDataAction::LoadSurrogateDataAction(InputParameters params) : Actio
 void
 LoadSurrogateDataAction::act()
 {
-  std::vector<SurrogateModel *> models;
-  _app.theWarehouse().query().condition<AttribSystem>("UserObject").queryInto(models);
-  for (auto model_ptr : models)
-    if (model_ptr->isParamValid("filename"))
```

Comment:
@rwcarlsen queryIntro doesn't seem to handle the dynamic_cast correctly. If I do what is here then the combined modules app fails the cast inside of TheWarehouse::queryInto